packages/dds/tree/docs/main/merge-semantics.md

```diff
@@ -0,0 +1,519 @@
+This matters because if the removal is undone, the subtree that is reintroduced will carry those modifications.
+
+These merge semantics effectively make removal akin to a move whose destination is an abstract "removed" or "recycling bin" location.
+This is in tune with the return value of the `treeStatus()` API which will return `TreeStatus.Removed` in that situation.
+
+#### Last Write Wins
+
+It's possible for concurrent edits to represent fundamentally incompatible user intentions.
+Whenever that happens, the edit that is sequenced last will win.
+
+Example 1:
+Alice and Bob concurrently change the background color of the same sticky note
+such that Alice changes it from yellow to red, and Bob changes it from yellow to blue.
+If the edits are sequenced such that Alice's edit is applied first and Bob's edit is applied second,
+then the background color of the note will change from yellow to red and then from red to blue.
+If the edits are sequenced in the opposite order,
+then the background color of the note will change from yellow to blue and then from blue to red.
+
+Example 2:
+Alice and Bob concurrently move the same sticky note
+such that Alice moves it from location X to location A, and Bob moves it from location X to location B.
+If the edits are sequenced such that Alice's edit is applied first and Bob's edit is applied second,
+then the note will first be moved from X to A and then from A to B.
+If the edits are sequenced in the opposite order,
+then the note will first be moved from X to B and then from B to A.
+Note that, because we treat [removal as movement](#removal-is-movement),
+this is true even when removals are involved:
+if the removal is sequenced last, then the node will be moved and then removed.
+If the move is sequenced last, then the node will be removed and then moved.
+
+### Non-Disjunctive Postconditions
+
+Disjunctive postconditions are postconditions of the form `<A> or <B> or ...`.
+None of `SharedTree`'s currently supported edits are of this form,
+which means they each have a single effect as opposed to one of many effects.
+
+For example, the postcondition for our implementation of move
+(that the moved node will reside at the chosen destination)
+is not predicated on the node not having been concurrently removed or moved.
+This means that when it comes to the final location of the node that is targeted by the move,
+there is only one possible outcome
+(the node being at the chosen destination)
+so long as the preconditions of the move were met.
+
+Note that such conditional postconditions would be different from additional preconditions.
+For example, adding a precondition that the moved node must not have been concurrently removed would,
+in a situation where that did occur,
+render such a move _and the whole transactions it figures in_ invalid.
+By contrast, making the postcondition conditional on the node not having been concurrently removed would simply render the move ineffective
```

Comment:
You introduce a definition of disjunctive postconditions above but never use it again in the examples. Perhaps illustrating how this example maps to the above A | B | ... form would make that clearer.